config/application.rb

```diff
@@ -8,8 +8,8 @@
 require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
-require "action_mailbox/engine"
-require "action_text/engine"
+# require "action_mailbox/engine"
+# require "action_text/engine"
```

Comment:
Is it worth deleting these rather than commenting them out?

Reply:
That's a good point. In most cases I'd agree but Rails apps seem to have a convention of using comments here rather than deleting lines.